tests/MailgunTest.php

```diff
@@ -49,4 +49,22 @@ public function testVerifyWebhookEmptyRequest()
         $postData = [];
         assert(!$client->verifyWebhookSignature($postData));
     }
+
+    public function testGetAttachmentOk()
+    {
+        $attachmentUrl = 'http://example.com';
+        $client = new Mailgun('key-3ax6xnjp29jd6fds4gc373sgvjxteol0');
+        $response = $client->getAttachment($attachmentUrl);
+
+        $this->assertInstanceOf('stdClass', $response);
+        $this->assertEquals($response->http_response_code, 200);
+    }
+
+    public function testGetAttachmentFail()
+    {
+        $this->setExpectedException('\\Mailgun\\Connection\\Exceptions\\GenericHTTPError');
+        $attachmentUrl = 'https://api.mailgun.net/non.existing.uri/1/2/3';
+        $client = new Mailgun('key-3ax6xnjp29jd6fds4gc373sgvjxteol0');
```

Comment:
@DavidGarciaCat of course it is not my key. If you look a few lines up in the file then you can see the same key (line 26: https://github.com/mailgun/mailgun-php/blob/master/tests/MailgunTest.php#L26).

Also mailgun documentation uses this key in examples.